client/web/src/enterprise/own/admin-ui/OwnAnalyticsPage.tsx

```diff
@@ -40,25 +40,22 @@ export const OwnAnalyticsPage: FC = () => {
 const OwnAnalyticsPanel: FC = () => {
     const { data, loading, error } = useQuery<GetInstanceOwnStatsResult>(GET_INSTANCE_OWN_STATS, {})
 
-    if (!data?.instanceOwnershipStats?.totalFiles) {
-        return <>{error && <ErrorAlert prefix="Error getting own analytics" error={error} />}</>
-    }
+    const totalFiles = data?.instanceOwnershipStats.totalFiles || 0
+    const totalCodeownedFiles = data?.instanceOwnershipStats.totalCodeownedFiles || 0
+    const totalAssignedOwnershipFiles = data?.instanceOwnershipStats.totalAssignedOwnershipFiles || 0
+    const totalOwnedFiles = data?.instanceOwnershipStats.totalOwnedFiles || 0
 
-    const totalFiles = data.instanceOwnershipStats.totalFiles
-    const totalCodeownedFiles = data.instanceOwnershipStats.totalCodeownedFiles
-    const totalAssignedOwnershipFiles = data.instanceOwnershipStats.totalAssignedOwnershipFiles
-    const totalOwnedFiles = data.instanceOwnershipStats.totalOwnedFiles
-
-    const totalCodeownedFilesPercent = Math.round((totalCodeownedFiles / totalFiles) * 100)
-    const totalAssignedOwnershipFilesPercent = Math.round((totalAssignedOwnershipFiles / totalFiles) * 100)
-    const totalOwnedFilesPercent = Math.round((totalOwnedFiles / totalFiles) * 100)
+    // Use Math.max(totalFiles, 1) to make sure we do not divide by 0.
+    const totalCodeownedFilesPercent = Math.round((totalCodeownedFiles / Math.max(totalFiles, 1)) * 100)
+    const totalAssignedOwnershipFilesPercent = Math.round((totalAssignedOwnershipFiles / Math.max(totalFiles, 1)) * 100)
+    const totalOwnedFilesPercent = Math.round((totalOwnedFiles / Math.max(totalFiles, 1)) * 100)
```

Comment:
nit: feel free to skip it if seems unnecessary
```suggestion
    const totalFilesDivisor = Math.max(totalFiles, 1)
    const totalCodeownedFilesPercent = Math.round((totalCodeownedFiles / totalFilesDivisor) * 100)
    const totalAssignedOwnershipFilesPercent = Math.round((totalAssignedOwnershipFiles / totalFilesDivisor) * 100)
    const totalOwnedFilesPercent = Math.round((totalOwnedFiles / totalFilesDivisor) * 100)
```